### PR TITLE
Delete Exam Functionality

### PR DESCRIPTION
**Pull request by akroon3r**
During testing, it was found out that the bookings feature lacked a function to delete an exam, not just the exam's booking. The exam-inventory table was modified to have a 'delete exam' dropdown action, which opens a confirmation modal. This modal displays basic exam information to the user before the user deletes the exam. This function has been tested using the CSR, GA and LIAISON roles.

### CODE REVIEW

File: frontend/src/exams/add-exam-modal.vue [hunk 1]
```diff
@@ -82,7 +82,7 @@
                  align-h="center"
                  align-content="center">
             <b-col>
-              <p><h5>Success.  Exam Details Added.</h5></p>
+              <p><h5>Success.  Exam Details Added.</h5>
```
Comment (sjrumsby) [inline frontend/src/exams/add-exam-modal.vue]: Some funky HTML going on here.

Comment (akroon3r) [reply frontend/src/exams/add-exam-modal.vue]: Yea so when I looked at this file pycharm said that p tag needed to be removed. I can re-add it, but it was showing up as an error for some reason?

Comment (akroon3r) [reply frontend/src/exams/add-exam-modal.vue]: screen shot 2019-02-27 at 11 23 33 am

Comment (akroon3r) [reply frontend/src/exams/add-exam-modal.vue]: ^^ just for evidence purposes

Comment (akroon3r) [reply frontend/src/exams/add-exam-modal.vue]: fixed, pushed up

**Pull request merged**